### PR TITLE
Add getPromptJSON utility functions for a third batch of widgets

### PR DESCRIPTION
**Pull request by MikeKlemarewski**
## Summary:
Adds  getPromptJSON utility functions for the following widgets:

- Matrix
- Measurer
- NumberLine
- Orderer
- PhET simulation
- Plotter
- Python program
- Sorter
- Video

Issue: TUT-930

## Test plan:
- Make sure tests pass
- Look at the tests and think if we should add any more edge cases

### CODE REVIEW

File: packages/perseus/src/widget-ai-utils/matrix/matrix.test.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    it("Should get prompt json which matches the state of the UI", async () => {
+        // Arrange
+        const {renderer} = renderQuestion(question);
+        const widget = renderer.getWidgetInstance("matrix 1");
+
+        // Act
+        const row1 = ["1", "2", "3", "4"];
+        const row2 = ["5", "6", "7", "8"];
+        const row3 = ["9", "10", "11", "12"];
+        const userInput = [...row1, ...row2, ...row3];
```
Comment (jeremywiebe) [inline packages/perseus/src/widget-ai-utils/matrix/matrix.test.ts:54]: This makes a single-dimension array for user-input? (ie `["1", "2", "3", "4", "5", "6"...]`)

Is that right?

File: packages/perseus/src/widget-ai-utils/orderer/prompt-utils.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import type orderer from "../../widgets/orderer/orderer";
+import type React from "react";
+
+export type OrdererPromptJSON = {
+    type: "orderer";
+    options: {
+        options: string[];
+    };
+    userInput: {
+        values: ReadonlyArray<string>;
```
Comment (jeremywiebe) [inline packages/perseus/src/widget-ai-utils/orderer/prompt-utils.ts:11]: **nit**: Can we use `ReadonlyArray<>` here?

**Review by jeremywiebe — APPROVED** [review]
This is looking great Mike.  I'd give the [same general feedback](https://github.com/Khan/perseus/pull/1822#pullrequestreview-2416438426) for this PR (just the two items of avoiding `as` and defining the expected JSON value explicitly in test assertions).

Thanks for all the great tests!

File: packages/perseus/src/widget-ai-utils/orderer/prompt-utils.ts (hunk 2 above)
Comment (MikeKlemarewski) [reply packages/perseus/src/widget-ai-utils/orderer/prompt-utils.ts:11]: Do you mean to use `options: ReadonlyArray<string>` above?

Comment (jeremywiebe) [reply packages/perseus/src/widget-ai-utils/orderer/prompt-utils.ts:11]: Yes, exactly. :)

File: packages/perseus/src/widget-ai-utils/matrix/matrix.test.ts (hunk 1 above)
Comment (MikeKlemarewski) [reply packages/perseus/src/widget-ai-utils/matrix/matrix.test.ts:54]: Yes. Below I was just iterating over each textbox in the UI to type something in. That maps nicely to a single dimension array.
